source/agora/test/Base.d

```diff
@@ -81,6 +87,13 @@ private UnitTestResult customModuleUnitTester ()
         if (mod.name.startsWith("agora") ||
             mod.name.startsWith("scpd"))
         {
+            if (filter.length > 0 &&
+                !canFind(mod.name.toLower(), filter))
```

Comment:
You see that's one thing I hate about ranges: the "copy is an implicit save" thing is nice, but it breaks ref / non copyable range.

In this case, I believe, this would be better / more correct:
```suggestion
                !canFind(mod.name.toLower(), filter.save))
```

Reply:
I thought it returned an array. Oh well.